.github/workflows/development.yml

```diff
@@ -0,0 +1,72 @@
+name: PR verification
+# -------------------------------------------------------------
+# This workflow will build and verify pull requests. It will:
+# - Build the branch and fail if the build fails
+# - Run code coverage tests on the PR branch
+# - Lint the PR branch
+# -------------------------------------------------------------
+
+on:
+    merge_group:
+    push:
+        branches: [main]
+    pull_request:
+        branches: [main]
+        types:
+            - opened
+            - synchronize
+            - reopened
+            - labeled
+            - unlabeled
+            - auto_merge_enabled
+
+permissions:
+    contents: read
+    pull-requests: write
+
+concurrency:
+    group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+    cancel-in-progress: true
+
+defaults:
+    run:
+        shell: bash
+
+jobs:
+    # -------------------------------------------------------------
+    # Validate build for various environments
+    # -------------------------------------------------------------
+    verify_builds:
+        name: Verify build
+        # Check that the PR is not in draft mode (or if it is, that it has the run_ci label to force a build)
+        if: ${{ github.event.pull_request.draft != 'true' || contains(github.event.pull_request.labels.*.name, 'run_ci') }}
+        strategy:
+            fail-fast: false
+            matrix:
+                system:
+                    - macos-latest
+                    - ubuntu-latest
+                    # - windows-latest
```

Comment:
Sadly the build fails pretty terrifically on windows. I tried to fix a few pathing issues in the scripts but the work kept multiplying and new errors cropping up so I abandoned that.

Reply:
Ooof, thanks for looking into it. Let’s leave it as-is for now, but make sure we build Swan working in Windows.